packages/core/admin/admin/src/content-manager/components/BlocksEditor/hooks/useBlocksStore.js

```diff
@@ -165,10 +165,34 @@ const handleBackspaceKeyOnList = (editor, event) => {
   const [currentListItem, currentListItemPath] = Editor.parent(editor, editor.selection.anchor);
   const [currentList, currentListPath] = Editor.parent(editor, currentListItemPath);
   const isListEmpty = currentList.children.length === 1 && currentListItem.children[0].text === '';
+  const isNodeStart = Editor.isStart(editor, editor.selection.anchor, currentListItemPath);
+  const isEditorStart = Editor.before(editor, currentListPath) == null;
+  const isFocusAtTheBeginningOfALine =
+    editor.selection.focus.offset === 0 && editor.selection.focus.at(-1) === 0;
```

Comment:
the code I gave earlier was off the top of my head, please test it before pushing

```suggestion
    editor.selection.focus.offset === 0 && editor.selection.focus.path.at(-1) === 0;
```